src/Controller/GeocoderController.php

```diff
@@ -1,8 +1,4 @@
 <?php
-/**
- * @file
- * Contains \Drupal\geocoder_autocomplete\Controller\GeocoderController.
- */
```

Comment:
Ok. Didi it. You are right. last phpcs changed specifications on namespaced classes and file doc comments. Going to merge ...